src/udisksdaemon.c

```diff
@@ -267,6 +289,8 @@ udisks_daemon_constructed (GObject *object)
   UDisksDaemon *daemon = UDISKS_DAEMON (object);
   GError *error;
   gboolean ret = FALSE;
+  gchar uuid_buf[37] = {0};
```

Comment:
Fixed.

Reply:
Ok, this wasn't the best advice, `UUID_STR_LEN` isn't defined in older versions of libuuid. You should probably unfix this :-)

```
udisksdaemon.c:292:18: error: 'UUID_STR_LEN' undeclared (first use in this function)
```

But I still think it would be better to use some constant than a "random" magic number. So maybe define the constant if it's not defined by libuuid?

Reply:
Good catch, unfixed by autotools magic.